MarkBind/markbind
# Fix annotation warnings in GitHub Actions

**What is the purpose of this pull request?**




**Overview of changes:**
Fixes #2040

- update CI script
- update docs to reflect this change

Left with the macOS warning, which should be fine as it doesn't really concern us and should be gone by Dec 1.
![image](https://user-images.githubusercontent.com/41845017/204736360-22b7adf4-af54-4f46-9a1e-fdcf2718db04.png)

**Anything you'd like to highlight/discuss:**


**Testing instructions:**
- verify action output warning area https://github.com/MarkBind/markbind/actions/runs/3581313613
- check updated docs https://deploy-preview-2041--markbind-master.netlify.app/userguide/deployingthesite



**Proposed commit message: (wrap lines at 72 characters)**
Fix annotation warnings in GitHub Actions


---

**Checklist:** :ballot_box_with_check:
